docs/how-to/required-software.md

```diff
@@ -0,0 +1,96 @@
+
+Select the latest stable version of Xcode (i.e Xcode 15.0.1, not the Beta version), then click `Download Xcode XX.X.X (2.96GB)` to download a compressed file:
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot of the Apple Developer website to select the version of Xcode](assets/required-software/xcode-website-selection.png)
+
+</div>
+
+Select a location to save the file:
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing the desired save location for Xcode](assets/required-software/download-xcode.png)
+
+</div>
+
+Once Xcode has downloaded, go to the folder where you saved the download (`Downloads` by default) and double click on `Xcode_XX.X.XX.xip` to extract the contents from the compressed file:
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Finder extracting Xcode from the compressed download](assets/required-software/xcode-extraction.png)
+
+</div>
+
+Once complete, you can launch Xcode from that location, however you can move it to the Applications if you desire by dragging Xcode to Applications (on the left hand side of Finder), however this may require Administrator permission:
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Finder extracting Xcode from the compressed download](assets/required-software/finder-admin-permissions.png)
+
+</div>
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Finder extracting Xcode from the compressed download](assets/required-software/xcode-in-applications.png)
+
+</div>
+
+
+When launching Xcode for the first time, you might be prompted to enter the administrator's name and password to continue:
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Xcode is being downloaded](assets/required-software/xcode-admin.png)
+
+</div>
+
+You'll need to accept the terms and conditions that Apple present in order for Xcode to run:
+
+<div class="filter: drop-shadow(0px 0px 20px rgba(0, 0, 0, 0.3)); margin: 60px 0; width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Xcode's Terms and Conditions](assets/required-software/xcode-terms.png)
+
+</div>
+
+A dialog will be presented showing which additional simulators and SDKs you want to install. Check the box for iOS, watchOS and tvOS can remain unchecked: 
```

Comment:
Just to make this a bit easier to read, can this last sentence be changed to say 
"Check the box for iOS (watchOS and tvOS can remain unchecked):"